### PR TITLE
Yay the challenge is complete

### PR DESCRIPTION
**Pull request by claalmve**
Any comments, discussion, advice, code snippets, etc. are welcome! Thanks for reading :)

Woohoo

### CODE REVIEW

File: day100/hi.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hurray it's been 100 days!
```
Comment (alberthu16) [inline day100/hi.txt]: asfwpef

Comment (stella-lu) [inline day100/hi.txt]: has it really?

**Comment by orrblue** [comment]
I think it's perf

**Comment by alberthu16** [comment]
Fix #46

**Comment by alberthu16** [comment]
Redundant work as #45

**Comment by alberthu16** [comment]
Fixes #40

**Pull request merged**